### DISCUSSION TITLE
GetSourceList in 5.0.0 beta1 version

### DISCUSSION THREAD

**Discussion by mititelud** [Palakis/obs-websocket#957] [General]
Hello,

I have installed the obs websocket 5.0.0 beta1 version in order to use the EventSubscription::InputVolumeMeters functionality which is currently not available in the released version 4.9.1.
What I intend to do is to get the sources of the OBS audio mixer and display the levels of the volume (vu meters) inputs of each source in my project. Please see the attached image, by vu meters I am reffering to the green filled rectangles of the mic

Using the obs websocket 4.9.1, I am currently retrieving the obs audix mixer sources using a GetSourcesList request but I have noticed that on 5.0.0 beta1 is not yet implemented according to the document below
https://docs.google.com/spreadsheets/d/1LfCZrbT8e7cSaKo_TuPDd-CJiptL7RSuo8iE63vMmMs/edit#gid=1232838276
Is there anything else that I could use in order to retrieve those sources using obs websocket 5.0.0 beta1?

Please address the following questions:
1. Do you have a 5.0.0 branch that could support the 4.9.1s GetSourcesList functionality and 
the EventSubscription::InputVolumeMeters functionality?
2. Do you have a release date for the 5.0.0 version or an estimate on when its going to be released? 

Thank you
![obs-mic](https://user-images.githubusercontent.com/6542142/176634667-ef9e247f-0f24-4658-8bd3-0413a4c620b0.png)

**Comment by tt2468** [comment]
GetSourceList in the sheet was meant to provide a list of inputs and scenes. The replacement for 4.x GetSourcesList in 5.x is GetInputList.

**Reply by mititelud** [reply]
Thank you for the quick reply
